repos/fdbt-site/src/pages/home.tsx

```diff
@@ -82,6 +83,23 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): {} => {
         redirectTo(ctx.res, '/multipleOperators');
     }
 
+    if (sessionNoc) {
+        const services = await getAllServicesByNocCode(sessionNoc);
+        const hasBodsServices = services.some((service) => service.dataSource && service.dataSource === 'bods');
+        const tndsServices = services.filter((service) => service.dataSource && service.dataSource === 'tnds');
+
```

Comment:
There are no unit tests on this PR - we need to add some.

Reply:
added changes